packages/rum-core/src/boot/rumPublicApi.ts

```diff
@@ -203,14 +203,24 @@ export function makeRumPublicApi(
     setUser: monitor((newUser: User) => {
       const sanitizedUser = sanitizeUser(newUser)
       if (sanitizedUser) {
-        user = sanitizedUser
+        userContextManager.set(sanitizedUser)
       } else {
         display.error('Unsupported user:', newUser)
       }
     }),
 
+    getUser: monitor(() => deepClone(userContextManager.get())),
+
     removeUser: monitor(() => {
-      user = {}
+      userContextManager.set({})
+    }),
+
+    addUserAttribute: monitor((attribute, value) => {
+      userContextManager.add(attribute, value)
+    }),
+
+    removeUserAttribute: monitor((attribute) => {
+      userContextManager.remove(attribute)
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏I think it's worth having tests to cover these new API